diff --git a/package/fenrirWeb/Config.in b/package/fenrirWeb/Config.in
new file mode 100644
index 0000000000..a10abdb8c9
--- /dev/null
+++ b/package/fenrirWeb/Config.in
@@ -0,0 +1,14 @@
+config BR2_PACKAGE_FENRIRWEB
+        bool "fenrirWeb"
+        depends on BR2_PACKAGE_PYTHON3
+        select BR2_PACKAGE_PYTHON_FENRIRCORE
+        select BR2_PACKAGE_PYTHON_BOTTLE
+        select BR2_PACKAGE_PYTHON_REQUESTS
+        select BR2_PACKAGE_PYTHON_PYROUTE2
+        select BR2_PACKAGE_PYTHON_SCAPY
+        select BR2_PACKAGE_PYTHON_JINJA2
+        help
+		"Fenrir provides a user friendly way to route all traffic from configured trough a VPN tunnel."
+		"FenrirWeb is a webinterface for viewing/ configuring fenrir." 	
+
+comment "FenrirWeb needs python3 and nginx(incl uwsgi module)"
diff --git a/package/fenrirWeb/fenrirWeb.mk b/package/fenrirWeb/fenrirWeb.mk
new file mode 100644
index 0000000000..fac9ac77fd
--- /dev/null
+++ b/package/fenrirWeb/fenrirWeb.mk
@@ -0,0 +1,8 @@
+FENRIRWEB_VERSION = 0.5.0
+FENRIRWEB_SETUP_TYPE = setuptools
+FENRIRWEB_SOURCE = fenrirWeb-$(FENRIRWEB_VERSION).tar.gz
+FENRIRWEB_SITE = https://files.pythonhosted.org/packages/b4/66/a97d4fbf237918a80324a6e9ad2957a22e16b442fc926a13ffa50936e574
+FENRIRWEB_LICENSE = GPL-2
+FENRIRWEB_LICENSE_FILES = LICENSE
+
+$(eval $(python-package))
